spec/jobs/tracking_sheet_report_job_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TrackingSheetReportJob, type: :job do
+  let(:druids) { ['druid:cc111dd2222'] }
+  let(:groups) { [] }
+  let(:user) { instance_double(User, to_s: 'amcollie') }
+  let(:output_directory) { bulk_action.output_directory }
+  let(:bulk_action) do
+    create(
+      :bulk_action,
+      action_type: 'TrackingSheetReportJob',
+      log_name: 'tmp/tracking_sheet_report_job_log.txt'
+    )
+  end
+  let(:log_buffer) { StringIO.new }
+
+  before do
+    allow(subject).to receive(:bulk_action).and_return(bulk_action)
+    allow(Ability).to receive(:new).and_return(ability)
+  end
+
+  after do
+    FileUtils.rm_rf(output_directory) if Dir.exist?(output_directory)
+  end
+
+  describe '#perform_now' do
+    context 'with authorization' do
+      let(:ability) { instance_double(Ability, can?: true) }
```

Comment:
I don't see your job checking authorization anywhere, but you're testing it here?

Reply:
Copy pasta, will remove.